### PR TITLE
Funcotator - Added WDL

### LINKED ISSUE #4065: Funcotator - add wdl / workflow
Add in a wdl / workflow for Funcotator.

### PR DESCRIPTION
**Pull request by jonn-smith**
Added a workflow/WDL for Funcotator.

Fixes #4065

### CODE REVIEW

File: scripts/mutect2_wdl/unsupported/funcotator.wdl [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    File ref_fasta
+    File ref_fasta_index
+    File ref_dict
+    File variant_vcf_to_annotate
+    String reference_version
+    String output_vcf_name
+
+    String? data_sources_tar_gz
+    String? transcript_selection_mode
+    Array[String]? transcript_selection_list
```
Comment (LeeTL1220) [inline scripts/mutect2_wdl/unsupported/funcotator.wdl]: Please file an issue to make this a `File` that is a list of transcripts.

File: scripts/mutect2_wdl/unsupported/funcotator.wdl [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    File? data_sources_tar_gz
+    String? transcript_selection_mode
+    Array[String]? transcript_selection_list
+    Array[String]? annotation_defaults
+    Array[String]? annotation_overrides
+
+    # ==============
+    # Process input args:
+    String transcript_selection_arg = if defined(transcript_selection_list) then "--transcript-list " else ""
+    String annotation_def_arg = if defined(annotation_defaults) then "--annotation-default " else ""
```
Comment (LeeTL1220) [inline scripts/mutect2_wdl/unsupported/funcotator.wdl]: Put spaces before the `--` in all three of these.

File: scripts/mutect2_wdl/unsupported/funcotator.wdl (hunk 1 above)
Comment (jonn-smith) [reply scripts/mutect2_wdl/unsupported/funcotator.wdl]: Filed as issue #4090

File: scripts/mutect2_wdl/unsupported/funcotator.wdl (hunk 2 above)
Comment (jonn-smith) [reply scripts/mutect2_wdl/unsupported/funcotator.wdl]: Fixed!

**Pull request merged**